src/main/java/tatracker/logic/parser/ParserUtil.java

```diff
@@ -288,4 +289,30 @@ public static Tab parseTabName(String tabName) throws ParseException {
         }
         return Tab.getTab(trimmedType);
     }
+
+    /**
+     * Parses and returns the pay rate specified by the user in the setrate command
+     *
+     * @param rate user input
+     * @return the pay rate specified by the user
+     * @throws ParseException invalid pay rate
+     */
+    public static int parseRate(String rate) throws ParseException {
+        requireNonNull(rate);
+        String trimmedRate = rate.trim();
+
+        if (!StringUtil.isUnsignedInteger(trimmedRate)) {
```

Comment:
String util has a non zero unsigned integer? So no need the following try catch?

Reply:
I thot this was alright just now?

Reply:
No my bad this is correct